dask/array/tests/test_array_core.py

```diff
@@ -3880,3 +3880,12 @@ def test_auto_chunks_h5py():
             with dask.config.set({'array.chunk-size': '1 MiB'}):
                 x = da.from_array(d)
                 assert x.chunks == ((256, 256, 256, 232), (512, 488))
+
+
+def test_no_warnings_from_blockwise():
+    x = da.ones((15, 15), chunks=(5, 5))
+
+    with warnings.catch_warnings(record=True) as record:
```

Comment:
Do we prefer `warnings.catch_warnings` here, or `pytest.warns`? Either way we need `assert not record`.

I would recommend `pytest.warns(None)`, since it'll be more consistent with cases where we want to assert something about the warning.